environment/src/main/java/com/sequenceiq/environment/environment/flow/deletion/EnvDeleteActions.java

```diff
@@ -211,6 +212,23 @@ protected void doExecute(CommonContext context, EnvDeleteFailedEvent payload, Ma
         };
     }
 
+    @Bean(name = "UMS_RESOURCE_DELETE_STARTED_STATE")
+    public Action<?, ?> umsResourceDeleteAction() {
+        return new AbstractEnvDeleteAction<>(EnvDeleteEvent.class) {
+
+            @Override
+            protected void doExecute(CommonContext context, EnvDeleteEvent payload, Map<Object, Object> variables) {
+                EnvironmentStatus environmentStatus = EnvironmentStatus.UMS_RESOURCE_DELETE_IN_PROGRESS;
+                ResourceEvent resourceEvent = ResourceEvent.ENVIRONMENT_UMS_RESOURCE_DELETION_STARTED;
+                EnvDeleteState envDeleteState = EnvDeleteState.UMS_RESOURCE_DELETE_STARTED_STATE;
+                String logDeleteState = "Assigned UMS resources";
```

Comment:
@oleewere are you sure that this message is correct?

Reply:
I suggest using lowercase except for proper names and abbreviations; so "assigned" instead of  "Assigned". This string will be inserted into the log message in line 243 below.